packages/plugin-ext-vscode/src/node/plugin-vscode-directory-handler.ts

```diff
@@ -58,6 +58,10 @@ export class PluginVsCodeDirectoryHandler implements PluginDeployerDirectoryHand
             types.push(PluginDeployerEntryType.FRONTEND);
         }
         context.pluginEntry().accept(...types);
+        if (types.length === 0) {
+            // Language server extension
+            context.pluginEntry().accept(PluginDeployerEntryType.BACKEND);
+        }
```

Comment:
Rather than have to `accept` twice we can implement a fallback like so?

```ts
async handle(context: PluginDeployerDirectoryHandlerContext): Promise<void> {
    await this.copyDirectory(context);
    const types: PluginDeployerEntryType[] = [];
    const packageJson: PluginPackage = context.pluginEntry().getValue('package.json');
    if (packageJson.browser) {
        types.push(PluginDeployerEntryType.FRONTEND);
    }
    if (packageJson.main || !packageJson.browser) {
        types.push(PluginDeployerEntryType.BACKEND);
    }
    context.pluginEntry().accept(...types);
}
```

Reply:
I like it :+1: